core/src/foundations/ionic.vars.scss

```
@@ -939,6 +367,578 @@ $ion-primitives-pink-1100: var(--ion-primitives-pink-1100, #8b2248);
+$ion-bg-extended-violet-subtle-press-rgb: var(--ion-bg-extended-violet-subtle-press-rgb, 220, 209, 251);
+$ion-bg-extended-purple-base-default: var(--ion-bg-extended-purple-base-default, $ion-primitives-purple-800);
+$ion-bg-extended-purple-base-default-rgb: var(--ion-bg-extended-purple-base-default-rgb, 167, 55, 220);
+$ion-bg-extended-purple-base-press: var(--ion-bg-extended-purple-base-press, $ion-primitives-purple-1000);
+$ion-bg-extended-purple-base-press-rgb: var(--ion-bg-extended-purple-base-press-rgb, 129, 42, 170);
+$ion-bg-extended-purple-subtle-default: var(--ion-bg-extended-purple-subtle-default, $ion-primitives-purple-100);
+$ion-bg-extended-purple-subtle-default-rgb: var(--ion-bg-extended-purple-subtle-default-rgb, 249, 243, 254);
+$ion-bg-extended-purple-subtle-press: var(--ion-bg-extended-purple-subtle-press, $ion-primitives-purple-300);
+$ion-bg-extended-purple-subtle-press-rgb: var(--ion-bg-extended-purple-subtle-press-rgb, 233, 211, 250);
+$ion-bg-extended-magenta-base-default: var(--ion-bg-extended-magenta-base-default, $ion-primitives-magenta-900);
+$ion-bg-extended-magenta-base-default-rgb: var(--ion-bg-extended-magenta-base-default-rgb, 193, 60, 172);
+$ion-bg-extended-magenta-base-press: var(--ion-bg-extended-magenta-base-press, $ion-primitives-magenta-1100);
+$ion-bg-extended-magenta-base-press-rgb: var(--ion-bg-extended-magenta-base-press-rgb, 137, 43, 122);
+$ion-bg-extended-magenta-subtle-default: var(--ion-bg-extended-magenta-subtle-default, $ion-primitives-magenta-100);
+$ion-bg-extended-magenta-subtle-default-rgb: var(--ion-bg-extended-magenta-subtle-default-rgb, 253, 243, 251);
+$ion-bg-extended-magenta-subtle-press: var(--ion-bg-extended-magenta-subtle-press, $ion-primitives-magenta-300);
+$ion-bg-extended-magenta-subtle-press-rgb: var(--ion-bg-extended-magenta-subtle-press-rgb, 249, 212, 241);
+$ion-bg-extended-pink-base-default: var(--ion-bg-extended-pink-base-default, $ion-primitives-pink-900);
+$ion-bg-extended-pink-base-default-rgb: var(--ion-bg-extended-pink-base-default-rgb, 197, 48, 102);
+$ion-bg-extended-pink-base-press: var(--ion-bg-extended-pink-base-press, $ion-primitives-pink-1100);
+$ion-bg-extended-pink-base-press-rgb: var(--ion-bg-extended-pink-base-press-rgb, 139, 34, 72);
+$ion-bg-extended-pink-subtle-default: var(--ion-bg-extended-pink-subtle-default, $ion-primitives-pink-100);
+$ion-bg-extended-pink-subtle-default-rgb: var(--ion-bg-extended-pink-subtle-default-rgb, 254, 243, 245);
+$ion-bg-extended-pink-subtle-press: var(--ion-bg-extended-pink-subtle-press, $ion-primitives-pink-300);
+$ion-bg-extended-pink-subtle-press-rgb: var(--ion-bg-extended-pink-subtle-press-rgb, 250, 211, 220);
+$ion-text-default: var(--ion-text-default, $ion-primitives-neutral-1200);
+$ion-text-default-rgb: var(--ion-text-default-rgb, 36, 36, 36);
+$ion-text-subtle: var(--ion-text-subtle, $ion-primitives-neutral-1000);
+$ion-text-subtle-rgb: var(--ion-text-subtle-rgb, 59, 59, 59);
```

```suggestion
$ion-text-subtle-rgb: var(--ion-text-subtle-rgb, $ion-primitives-neutral-1000-rgb);
```

Should this fallback to `$ion-primitives-neutral-1000-rgb` (and the same for all other text rgb vars)?